engine/classes/Elgg/HooksRegistrationService.php

```diff
@@ -14,7 +14,19 @@
  */
 abstract class HooksRegistrationService {
 
-	private $handlers = array();
+	const REG_KEY_PRIORITY = 0;
+	const REG_KEY_INDEX = 1;
+	const REG_KEY_HANDLER = 2;
```

Comment:
Prefer these to be strings instead of ints, but the fact that they're constants eases most of the readability pain, so not a huge deal.